disk/datadog_checks/disk/disk.py

```diff
@@ -305,7 +305,10 @@ def _compile_tag_re(self):
         device_tag_list = []
         for regex_str, tags in self._device_tag_re.iteritems():
             try:
-                device_tag_list.append([re.compile(regex_str), [t.strip() for t in tags.split(",")]])
+                device_tag_list.append([
+                    re.compile(regex_str, re.IGNORECASE),
```

Comment:
I don't think you want to blanket ignore case.
On linux /Dev/Foo is different from /dev/foo
On Windows c: is the same as C:\